### PR TITLE
chore: add pull request template

### PR DESCRIPTION
**Pull request by LiangliangSui**

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<!--
+**Thanks for contributing to Fury.**
+
+**If this is your first time opening a PR on fury, you can refer to [CONTRIBUTING.md](https://github.com/apache/incubator-fury/blob/main/CONTRIBUTING.md).**
+
+Contribution Checklist
+
+    - The **Apache Fury (incubating)** community has restrictions on the naming of pr titles. You can also find instructions in [CONTRIBUTING.md](https://github.com/apache/incubator-fury/blob/main/CONTRIBUTING.md).
+
+    - Fury does not accept PRs that degrade performance. If the PR you submit will have an impact on performance, please Benchmark it yourself first.
```
Comment (chaokunyang) [inline .github/pull_request_template.md:10]:
```suggestion
    - Fury has a strong focus on performance. If the PR you submit will have an impact on performance, please benchmark it first and provide the benchmark result here.
```
I personly feel `Fury does not accept PRs that degrade performance` is a little aggressive.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+<!--
+If any user-facing interface changes, please [open an issue](https://github.com/apache/incubator-fury/issues/new/choose) describing the need to do so and update the document if necessary.
+-->
+
+
+## Benchmark
+
+<!--
+**Fury doesn't accept performance degradation**
```
Comment (chaokunyang) [inline .github/pull_request_template.md:39]: Could we remove this line? The community should be open, this should be case by case.

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+- #xxxx2
+-->
+
+
+## Does this PR introduce _any_ user-facing change?
+
+<!--
+If any user-facing interface changes, please [open an issue](https://github.com/apache/incubator-fury/issues/new/choose) describing the need to do so and update the document if necessary.
+-->
+
```
Comment (chaokunyang) [inline .github/pull_request_template.md:34]:
```suggestion
- [ ] Does this PR introduce any public API change?
- [ ] Does this PR introduce any binary protocol compatibility change?
```

File: .github/pull_request_template.md (hunk 1 above)
Comment (LiangliangSui) [reply .github/pull_request_template.md]: Updated.

File: .github/pull_request_template.md (hunk 3 above)
Comment (LiangliangSui) [reply .github/pull_request_template.md:34]: Updated.

File: .github/pull_request_template.md (hunk 2 above)
Comment (LiangliangSui) [reply .github/pull_request_template.md]: Updated.

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+<!--
+Is there any related issue? Please attach here.
+
+- #xxxx0
+- #xxxx1
+- #xxxx2
+-->
+
+
+## Does this PR introduce _any_ user-facing change?
```
Comment (chaokunyang) [inline .github/pull_request_template.md:29]:
```suggestion
## Does this PR introduce any user-facing change?
```

Comment (LiangliangSui) [reply .github/pull_request_template.md:29]: _any_(\_any\_) is in italics to emphasize importance, which I think is necessary

Comment (chaokunyang) [reply .github/pull_request_template.md:29]: I prefer not emphasize it, the items list here are all important

**Review by chaokunyang — APPROVED** [review]
LGTM, could you take a look at this @theweipeng @PragmaTwice

**Review by theweipeng — APPROVED** [review]

**Pull request merged**